SpatialGDK/Source/SpatialGDK/Private/EngineClasses/RPCContainer.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Any reason why you put this into the EngineClasses folder? The intent of EngineClasses are for classes that extend off of native Unreal classes: e.g. NetDriver, ActorChannel. I don't think this one does?

Reply:
RPCContainer is used in SpatialSender and SpatialReceiver. What if I move it to Interop instead?

Reply:
Yes. Please move it to Interop.

Reply:
Moved.